explorer/src/Pos/Explorer/ExplorerMode.hs

```diff
@@ -66,21 +66,21 @@ import           Test.Pos.Block.Logic.Mode (TestParams (..))
 -- testing (and running).
 type ExplorerMode ctx m =
     ( MonadDBRead m
-    -- ^ Database operations
+    --  Database operations
     , MonadSlots ctx m
-    -- ^ Slotting
+    --  Slotting
```

Comment:
@jcmincke What about using `*` to expose these comments in the documentation? http://haskell-haddock.readthedocs.io/en/latest/markup.html#documentation-structure-examples

Reply:
Inside tuple syntax -- | , -- ^ or -- * are not legal. -- *  is ok for "top level" documentation of types/functions/...

Reply:
Bad me! I genuinely thought I was staring at a top level module 😅